egs/reverb/s5/run.sh

```diff
@@ -33,305 +35,164 @@ fi
 fi
-export reverb_dt=$REVERB_home/REVERB_WSJCAM0_dt
-export reverb_et=$REVERB_home/REVERB_WSJCAM0_et
-export reverb_real_dt=$REVERB_home/MC_WSJ_AV_Dev
-export reverb_real_et=$REVERB_home/MC_WSJ_AV_Eval
-
-# set the directory of the multi-condition training data to be generated
-reverb_tr=`pwd`/data_tr_cut/REVERB_WSJCAM0_tr_cut
 
-# LDA context size (left/right) (4 is default)
-context_size=4
+#training set and test set
+train_set=tr_simu_8ch
+test_sets="dt_real_1ch dt_simu_1ch et_real_1ch et_simu_1ch dt_real_2ch_beamformit dt_simu_2ch_beamformit et_real_2ch_beamformit et_simu_2ch_beamformit dt_real_8ch_beamformit dt_simu_8ch_beamformit et_real_8ch_beamformit et_simu_8ch_beamformit dt_real_1ch_wpe dt_simu_1ch_wpe et_real_1ch_wpe et_simu_1ch_wpe dt_real_2ch_wpe dt_simu_2ch_wpe et_real_2ch_wpe et_simu_2ch_wpe dt_real_8ch_wpe dt_simu_8ch_wpe et_real_8ch_wpe et_simu_8ch_wpe dt_cln et_cln"
 
 # The language models with which to decode (tg_5k or bg_5k)
 lm="tg_5k"
 
 # number of jobs for feature extraction and model training
-nj_train=30
-
+nj=92
 # number of jobs for decoding
-nj_decode=8
-
-# set to true if you want the tri2a systems (re-implementation of the HTK baselines)
-do_tri2a=true
+decode_nj=10
 
-if [ $stage -le 1 ]; then
-  # Generate multi-condition training data
-  # Note that utterance lengths match the original set.
-  # This enables using clean alignments in multi-condition training (stereo training)
-  local/REVERB_create_mcdata.sh $wsjcam0 $reverb_tr
+wavdir=${PWD}/wav
+pesqdir=${PWD}/local
+if [ ${stage} -le 1 ]; then
+  # data preparation
+  echo "stage 0: Data preparation"
+  local/generate_data.sh --wavdir ${wavdir} ${wsjcam0}
+  local/prepare_simu_data.sh --wavdir ${wavdir} ${reverb} ${wsjcam0}
+  local/prepare_real_data.sh --wavdir ${wavdir} ${reverb}
 fi
 
 if [ $stage -le 2 ]; then
+  local/run_wpe.sh
+  local/run_beamform.sh ${wavdir}/WPE/
+fi
+
+# Compute dereverberation scores
+if [ $stage -le 3 ] && $compute_se; then
+  if [ ! -d local/REVERB_scores_source ] || [ ! -d local/REVERB_scores_source/REVERB-SPEENHA.Release04Oct/evaltools/SRMRToolbox ] || [ ! -f local/PESQ ]; then
+    # download and install speech enhancement evaluation tools
+    local/download_se_eval_tool.sh
+  fi
+  local/compute_se_scores.sh --nch $nch_se $reverb $wavdir $pesqdir
+  cat exp/compute_se_${nch_se}ch/scores/score_SimData
+  cat exp/compute_se_${nch_se}ch/scores/score_RealData
+fi
+
+if [ $stage -le 4 ]; then
   # Prepare wsjcam0 clean data and wsj0 language model.
   local/wsjcam0_data_prep.sh $wsjcam0 $wsj0
 
   # Prepare merged BEEP/CMU dictionary.
   local/wsj_prepare_beep_dict.sh
 
   # Prepare wordlists, etc.
-  utils/prepare_lang.sh data/local/dict "<SPOKEN_NOISE>" data/local/lang_tmp data/lang
+  utils/prepare_lang.sh data/local/dict "<NOISE>" data/local/lang_tmp data/lang
 
   # Prepare directory structure for clean data. Apply some language model fixes.
   local/wsjcam0_format_data.sh
+  
+  local/train_lms_srilm.sh \
```

Comment:
The REVERB challenge uses a pre-trained LM. This seems to train a LM only with transcriptions, which is not the same condition.

Reply:
Issue fixed and RESULTS updated.

Reply:
You should delete `local/train_lms_srilm.sh`, as it is not used.

Reply:
It's already be removed. This is the Outdated version. Thanks!